src/core/assets/NewChatIcon.tsx

```diff
@@ -0,0 +1,21 @@
+import { SVGProps } from 'react';
+
+const NewChatIcon = (props: SVGProps<SVGSVGElement>) => {
```

Comment:
NewChatIcon as a component. This could replace the ChatHistory svg code and be used in the navbar when that gets added